src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -802,7 +802,7 @@ static void window_editor_inventions_list_scrollpaint(rct_window *w, rct_drawpix
         {
             const rct_string_id rideGroupName = get_ride_naming(researchItem->baseRideType, get_ride_entry(researchItem->entryIndex)).name;
             format_string(groupNamePtr, sizeof(groupNameBuffer), STR_INVENTIONS_LIST_RIDE_AND_VEHICLE_NAME, (void*) &rideGroupName);
-            format_string(vehicleNamePtr, sizeof(vehicleNamePtr), itemNameId, nullptr);
+            format_string(vehicleNamePtr, sizeof(vehicleNameBuffer), itemNameId, nullptr);
```

Comment:
Thanks for the fix. `Util::CountOf` would have caught this mistake at compile time. Would you mind changing this and the other calls around here to use that function instead of `sizeof`?

Reply:
Sure, will do it now.

Reply:
Done 👍